go.mod

```diff
@@ -8,27 +8,29 @@ replace github.com/containers/image/v5 => github.com/containers/image/v5 v5.23.1
 // Pin moby/buildkit until docker/docker is upgraded
 replace github.com/moby/buildkit => github.com/moby/buildkit v0.11.6
```

Comment:
Can we unpin `buildkit` version afterwards?

Reply:
Yeah.